packages/module/src/ConsoleShared/src/components/spotlight/spotlight.scss

```diff
@@ -36,14 +36,14 @@
     mix-blend-mode: hard-light;
   }
   &__element-highlight-noanimate {
-    border: var(--pf-v5-global--BorderWidth--xl) solid var(--pf-v5-global--palette--blue-200);
-    background-color: var(--pf-v5-global--palette--black-500);
+    border: var(--pf-v6-global--BorderWidth--xl) solid var(--pf-v6-global--palette--blue-200);
+    background-color: var(--pf-v6-global--palette--black-500);
```

Comment:
For border-width, I'd probably use the strong or extra-strong border in this list https://github.com/patternfly/patternfly/blob/ceb0aa1a538064c928727d7051792bf07ca6ff78/src/patternfly/base/tokens/tokens-default.scss#L256-L257

For border-color, there are a bunch of tokens to choose from, so it may depend on the use case but I'm wondering if it should use `--pf-t--global--border--color--brand--default`

Depending on how this background is used, I'm not sure what the best token would be. The black color palette is here - https://github.com/patternfly/patternfly/blob/ceb0aa1a538064c928727d7051792bf07ca6ff78/src/patternfly/base/tokens/tokens-palette.scss#L63-L72

From the classname/z-index, I wonder if `--pf-t--global--background--color--backdrop--default` would work, which is a semi-opaque dark background?